camelComponent/README.md

```diff
@@ -1,7 +1,8 @@
 # Overview
```

Comment:
There were changes made to the component to deal with things better. These are good improvements to the Vantiq component -- things we discovered using the generated assemblies (or support we required).